MANIFEST.in

```diff
@@ -0,0 +1 @@
+include clinvar_ingest/cloud/bigquery/bq_json_schemas/*.json
```

Comment:
Actually... did you explore doing this with pyproject/setuptools?  I think we should be able to get the configuration in there...

Reply:
If I remove MANIFEST.in I cannot figure out how to get it to include those JSON files, even if they are specified in pyproject.toml. I find the docs very confusing to parse for all this packaging stuff but it seems like for putting the file pattern in both places is necessary https://setuptools.pypa.io/en/latest/userguide/datafiles.html

Unless you know of a way to omit MANIFEST.in and just use pyproject.toml

Reply:
I can take a peek... the docs and syntax for this is mindblowingly confusing.